plot_types/stats/barbs.py

```diff
@@ -14,14 +14,13 @@
 np.random.seed(1)
 X = [[2, 4, 6]]
 Y = [[1.5, 3, 2]]
-U = -np.ones((1, 3)) * 0
-V = -np.ones((1, 3)) * np.linspace(50, 100, 3)
+U = np.zeros_like(X)
+V = -np.ones_like(X) * np.linspace(50, 100, 3)
```

Comment:
Lower case?

Reply:
See comment in https://github.com/matplotlib/matplotlib/pull/20509#pullrequestreview-692294629